text/0000-project-goals.md

```diff
@@ -0,0 +1,196 @@
+* **Goals cover a problem, not a solution.** As much as possible, the goal should describe the problem to be solved, not the precise solution. This also implies that accepting a goal means the project is committing that the **problem** is a priority: we are not committing to accept any particular solution.
+* **Nothing good happens without an owner.** Rust endeavors to run an open, participatory process, but ultimately achieving any concrete goal requires someone (or a small set of people) to take ownership of that goal. Owners are entrusted to listen, take broad input, and steer a well-reasoned course in the tradeoffs they make towards implementing the goal. But this power is not unlimited: owners make proposals, but teams are ultimately the ones that decide whether to accept them.
+* **To everything, there is a season.** While there will be room for accepting new goals that come up during the year, we primarily want to pick goals during a fixed time period and use the rest of the year to execute.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The long-term vision is to create a sustainable goals process for the project. Per the axiom that **goals cover a problem, not a solution**, this RFC does not propose a specific process. **Rather, the goal is to devise the process.** To help us get going quickly, the intent is that the goal owner will design and drive an experiental process, including (a) selecting a slate of goals that will be confirmed by the teams they affect; (b) monitoring and reporting on progress towards those goals; and (c) developing infrastructure to support that monitoring and lessen the load. Experience from that will be used to shape an RFC that describes the process to use for the future (assuming the experiment is a success). 
+
+## Ownership and other resources
+
+**Owner:** nikomatsakis
+
+* nikomatsakis can commit 20% time (avg of 1 days per week) to pursue this task, which he estimates to be sufficient.
+
+### Support needed from the project
+
+* Project website resources to do things like
+    * post blog posts on both Inside Rust and the main Rust blog;
+    * create a tracking page (e.g., `https://rust-lang.org/goals`);
+    * create repositories, etc.
+* For teams opting to participate in this experimental run:
+    * they need to meet with the goal committee to review proposed goals, discuss priorities;
+    * they need to decide in a timely fashion whether they can commit the proposed resources
+
+## Outputs and milestones
+
+### Outputs
+
+There are three specific outputs from this process:
+
+* A **goal slate** for the second half of 2024, which will include
+    * a set of goals, each with an owner and with approval from their associated teams
+    * a high-level write-up of why this particular set of goals was chosen and what impact we expect for Rust
+* **Regular reporting** on the progress towards these goals over the course of the year
```

Comment:
Where does the goal progress get reported? One of the Project blogs? Should there be a T-Goals? Or is reporting to the relevant teams sufficient?

Reply:
I expect to post updates to one of the Rust blogs. My current plan is to produce monthly updates on Inside Rust that are built by 

* automatically scraping content from tracking issues
* adding a bit of extra commentary

along with three posts on the main Rust blog

* one announcing the slate
* one mid-way through describing progress
* one towards the end with a retro + (if applicable) new slate

Reply:
I tried to make this clearer in the text

Reply:
Sounds good! Out of curiousity, if the Council were to arrange to hire administrative support, would you be interested in having their help with the regular updates?